### PR TITLE
custom caption (required for fbia)

### PR DESCRIPTION
**Pull request by kesla**
R = @iefserge
T = minor

### CODE REVIEW

File: lib/components/embed.js [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-  test: ({embedType}) => embeds[embedType],
-  render: ({props}) => {
-    const {embedType, caption, figureProps} = props;
-    const embed = embeds[embedType] && embeds[embedType](props);
+const defaultRenderCaption = text => <figcaption>{text}</figcaption>;
 
-    const captionElm = (caption && caption.length > 0)
-      ? <FigureCaption items={caption} /> : '';
+const setup = ({embeds, customCaption}) => {
+  const renderCaption = customCaption || defaultRenderCaption;
```
Comment (iefserge) [inline lib/components/embed.js]: @kesla this assumes deku/jsx requirement on the module consumer side, right?

**Comment by iefserge** [comment]
1 question, LGTM

File: lib/components/embed.js (hunk 1 above)
Comment (kesla) [inline lib/components/embed.js]: yeah, I didn't know how to do this otherwise...

Comment (iefserge) [inline lib/components/embed.js]: ok

**Pull request merged**